pypesto/sample/pymc.py

```diff
@@ -18,6 +18,14 @@
 
 logger = logging.getLogger(__name__)
 
+# __all__ definition for Sphinx
```

Comment:
```suggestion
```

This is somewhat misleading, since `__all__` is not really specifically a sphinx thing but has broader implications.